### PR TITLE
Adding client file reloading and client test running to the manage.py script.

### PR DESCRIPTION
**Pull request by DavidEBest**
Previously, the runserver directive only watched for changes in the python files. Now it also watches the static files. Additionally, the client tests can now be run via the script.

### CODE REVIEW

**Comment by DavidEBest** [comment]
@catherinedevlin When you get a chance, could you review this? I'm stepping into python-land a bit. :)

File: manage.py [hunk 1]
```diff
@@ -46,9 +46,23 @@ def __call__(self, app, host, port, use_debugger, use_reloader,
         if use_reloader is None:
             use_reloader = use_debugger
 
+        extra_files = None
```
Comment (catherinedevlin) [inline manage.py]: None will throw an error when you try to append to it in line 60 - should be

    extra_files = []

**Comment by DavidEBest** [comment]
@catherinedevlin Thanks for the feedback. I've changed the initialization of the extra_files variable.

File: manage.py [hunk 2]
```diff
@@ -66,13 +80,30 @@ def _make_context():
...
+    """Run the tests."""
+    print type
+    if type == "server":
+        return server_test()
+    elif type == "client":
+        return client_test()
+    else:
+        server_exit_code = server_test()
+        client_exit_code = client_test()
+        return server_exit_code and client_exit_code
```
Comment (catherinedevlin) [inline manage.py]: Assuming this returns 0 on success (haven't tried it), should this be `server_exit_code or client_exit_code` to give the worst of the two?

**Pull request merged**